src/components/views/settings/feedback.tsx

```diff
@@ -1,4 +1,4 @@
-import React, {useState} from 'react';
+import React from 'react';
```

Comment:
Is this fle updated because you checked-out from an existing branch?

Reply:
Feedback screen changes are for https://github.com/project-vagabond/covid-green-app/issues/453#issuecomment-693529856 - it's not clear how it was written in the issue but it came up in the meeting yesterday that due to a mixup Antoine and Alex hadn't seen the draft designs for the new Feedback screen that Kevin etc supplied before we implemented them so that's the new version.